source/Databricks/source/SqlStatementExecution.IntegrationTests/Client/DatabricksStatementsTests.cs

```diff
@@ -168,9 +170,69 @@ public async Task ExecuteStatementAsync_WhenCancelled_IsCancelledDuringTheInitia
         task.IsCanceled.Should().BeTrue();
     }
 
+    [Fact]
+    public async Task Given_Cancellation_Token_When_Token_Is_Cancelled_Then_Cluster_Job_Is_Cancelled()
```

Comment:
The other tests seems to follow a 3 section naming pattern where underscore (_) is used to seperate each section.
Following that format I would expect the test method name to be:

```suggestion
    public async Task GivenCancellationToken_WhenTokenIsCancelled_ThenClusterJobIsCancelled()
```